### PR TITLE
[Product] Fix showing attributes for a product

### PR DESCRIPTION
**Pull request by GSadee**
Replaces #542

### CODE REVIEW

File: src/Factory/Product/ProductAttributeValueViewFactory.php [hunk 1]
```diff
@@ -18,39 +19,45 @@ public function __construct(string $productAttributeValueViewClass)
...
         /** @var ProductAttributeTranslationInterface $productAttributeTranslation */
-        $productAttributeTranslation = $productAttribute->getTranslation($locale);
+        $productAttributeTranslation = $productAttribute->getTranslation($localeCode);
         $productAttributeValueView->name = $productAttributeTranslation->getName();
 
         return $productAttributeValueView;
     }
 
-    private function resolveSelectAttribute(ProductAttributeValueInterface $productAttributeValue)
+    private function getAttributeValue(ProductAttributeValueInterface $productAttributeValue, string $localeCode)
```
Comment (mamazu) [inline src/Factory/Product/ProductAttributeValueViewFactory.php]: Can we type-hint this to an array?

Comment (lchrusciel) [reply src/Factory/Product/ProductAttributeValueViewFactory.php]: It may be string or array

Comment (mamazu) [reply src/Factory/Product/ProductAttributeValueViewFactory.php]: Ah okay so this method handles both cases? Choices and scalar values?

Comment (lchrusciel) [reply src/Factory/Product/ProductAttributeValueViewFactory.php]: Yes, we have too. One of the attribute types (multiple select) may return more than one value, therefore we will support mixin here. In the future, one can add JSON object for instance.

Comment (mamazu) [reply src/Factory/Product/ProductAttributeValueViewFactory.php]: Okay understandable.

**Comment by lchrusciel** [comment]
Closing in favor of #586

**Pull request closed without merging**